onebusaway-android/src/main/java/org/onebusaway/android/nav/NavigationService.java

```diff
@@ -64,6 +64,8 @@
 
     public static final String LOG_DIRECTORY = "ObaNavLog";
 
+    public static String LOG_FILE = null;
```

Comment:
Let's call this `mLogFile` instead, as that's the usual convention for class member variables.  Also, I don't believe this should be `static`?  If there is more than one service they should each have their own file to write too.  This doesn't happen in our current design but we should write the code to handle this.

Reply:
@barbeau  We are using the String variable "LOG_FILE" to store the device location of a trip log csv file. This variable is used in FeedbackActivity.java and FeedbackReceiver.java to locate the log and move to the 'yes' or 'no' device folders for upload to Firebase. The variable value will remain the same for the entire service. Can you please let me know if it is ok to continue with the existing variable name or should we change the name and access specifier?

Reply:
Oh, I missed that in my first pass just looking at the code in the PR.  That needs to change - it's not safe to depend on a variable in a Service from an Activity or Receiver - the Service could easily end and be destroyed before the user interacts with the notification to provide feedback, in which case LOG_FILE would be null.

The file name needs to be stored somewhere outside of the Service.  Because we only have one notification prompt for feedback at this point, I believe you could store it as an Intent extra when creating the notification, and then read it from the intent when you receive it in the BroadcastReceiver, and pass it into the Activity via an intent extra as well.

Reply:
We have moved the user feedback notification code to NavigationService.java so there is no need to access the log file name outside of the service and can be added as an intent extra.